src/lingua/extractors/__init__.py

```diff
@@ -1,6 +1,10 @@
 from __future__ import print_function
-from pkg_resources import DistributionNotFound
-from pkg_resources import working_set
+
+try:
+    from importlib.metadata import entry_points
+except ImportError:
+    from importlib_metadata import entry_points
```

Comment:
To prevent the same issue as with configparser above, it could be helpful to document, when this workaround is no longer needed.

Reply:
```suggestion
try:
    from importlib.metadata import entry_points
except ImportError:  # required up to Python 3.9
    from importlib_metadata import entry_points
```
At least AFAIU https://docs.python.org/3.9/library/importlib.metadata.html#entry-points and the "Compatibility Note" in https://docs.python.org/3.10/library/importlib.metadata.html#entry-points